src/Components/MainPage/MovieList/MovieList.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef, useEffect } from "react";
+import Movie from "./Movie";
+import "./movielist.css";
+import { Link } from "react-router-dom";
+
+const MovieList = ({ movies }) => {
+  const refField = useRef(null);
+
+  useEffect(() => {
+    if (movies.length > 0 && refField.current) {
+      refField.current.scrollIntoView({
+        behavior: "smooth",
+      });
+    }
+  }, [movies]);
+
+  return (
+    movies.length > 0 && (
+      <div className="search-results-wrapper">
+        <h1 className="search-results" ref={refField}>
+          Search Results
+        </h1>
+        <div className="movie-container">
+          {movies.map((movie, index) => {
+            return (
+              <Link to={`/movie/${movie.id}`} key={index} className="link">
```

Comment:
I would recommend not to use index as key, if you have other unique identifier. For example, in this case you can use `movie.id`.

Read more about how keys work:
https://reactjs.org/docs/lists-and-keys.html

And specifically about indexes:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318